### ISSUE TITLE
public.api.bsky.app/xrpc/app.bsky.feed.searchPosts "cursor" parameter is not working.

### ISSUE THREAD

**Issue by wiksien** [bluesky-social/bsky-docs#296]
I may be doing something stupid, so feel free to correct me, but I think the cursor parameter is broken on the API side of things.

Other parameters work like for example: https://public.api.bsky.app/xrpc/app.bsky.feed.searchPosts?q=test&limit=100

But when I add the cursor parameter I get 403 Forbidden, for example: https://public.api.bsky.app/xrpc/app.bsky.feed.searchPosts?q=test&limit=100&cursor=%2210%22

I want to use the cursor to fetch 400 posts instead of the limited 100.

For any sort of help I thank in advance!

**Comment by bnewbold** [comment]
Yeah, the Bluesky API server doesn't currently support enumerating additional posts with the `cursor` parameter for unauthenticated post search. We actually frequently disable public post search all-together, because bot nets have been hitting it too hard (despite rate-limits), causing downtime for regular users.

Technically, somebody else could implement this API endpoint, using the same Lexicon schema, and support `cursor` in all situations.

We will try to get the docs updated so this is less confusing.
